tests/unit/test_dashboards.py

```diff
@@ -509,90 +509,55 @@ def test_tile_places_tile_below():
     assert tile.position.y == 8
 
 
-def test_dashboards_saves_sql_files_to_folder(tmp_path):
```

Comment:
Moved the `save_to_folder` tests down to keep the test order somewhat consistent